shared/domain/src/main/java/com/ivy/domain/usecase/csv/ExportCsvUseCase.kt

```diff
@@ -74,7 +74,7 @@ class ExportCsvUseCase @Inject constructor(
         categoriesMap: Map<CategoryId, Category>,
     ): String = csvRow {
         // Date
-        csvAppend(date?.csvFormat())
+        csvAppend(date?.toString())
```

Comment:
This might break the CSV, I don't feel confident that it'll work

Reply:
For example some user Locales may export it with `,` which will result in broken CSV. What's the issue with .csvExport()?

Reply:
Also users prefer to see their CSV time in their local time (given that the local time is correct)

Reply:
Thanks for the feedback! I'll switch back to .csvFormat() and adjust it to export the time in the user's local time zone while keeping the format consistent and CSV-safe.

Reply:
> For example some user Locales may export it with `,` which will result in broken CSV. What's the issue with .csvExport()?

The .csvFormat() method was generating incorrect time values.

Reply:
Try fixing the .csvExport(). I'm conservative about small time fixes because they fix one thing and break another two. We should implement the proper solution that I started in #3396. This converter works properly but must be used everywhere and also edge-cases like Time.MIN/MAX aren't handled gracefully